CHANGELOG.md

```diff
@@ -23,11 +23,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add avifenc flag suffixes ":update" and ":u". Quality-relative,
   tiling-relative and codec-specific flags can now be positional, relative to
   input files.
+* Add experimental support for progressive AVIF encoding in avifenc.
+  A "layer" of progressive AVIF works like a frame of animated AVIF, but
+  rendered in progressive manner on supported viewers.
+  Only aom supports progressive AVIF encoding at the time of writing.
+  Add --layer flag to set number of layers of progressive AVIF.
+  Add --scaling-mode flag to set scaling mode of each layer.
```

Comment:
Since `--scaling-mode` can also be applied to non-progressive images and animations, it could have been added in a separate PR to make this review easier.